restricted_site_access.php

```diff
@@ -780,7 +780,17 @@ public static function page_cache_notice() {
 			?>
 			<div data-rsa-notice="page-cache" class="notice notice-error is-dismissible">
 				<p>
-					<strong><?php esc_html_e( 'Page caching appears to be enabled. Restricted Site Access may not work as expected. <a href="https://wordpress.org/plugins/restricted-site-access/#faq">Learn more</a>.', 'restricted-site-access' ); ?></strong>
+					<strong>
+					<?php
+						echo wp_kses_post(
+							sprintf(
+								/* translators: %s: https://wordpress.org/plugins/restricted-site-access/#faq */
+								__( 'Page caching appears to be enabled. Restricted Site Access may not work as expected. <a href="%s">Learn more</a>.' ),
+								'https://wordpress.org/plugins/restricted-site-access/#faq'
```

Comment:
Core makes the URL translatable (I'm guessing so translations could use their localized URL for the plugin repo).

How about we add this too?

```suggestion
								__( 'https://wordpress.org/plugins/restricted-site-access/#faq', 'restricted-site-access' )
```